eslint.config.mjs

```diff
@@ -0,0 +1,11 @@
+import globals from "globals";
```

Comment:
Can I get the ESLint configuration in something other than `.mjs`, preferably `.js`

Reply:
Fine. I'll force push and change that. Unless that's bad practice?

Reply:
Nvm will just commit another